ui/app/controllers/variables/variable/index.js

```diff
@@ -10,6 +11,18 @@ export default class VariablesVariableIndexController extends Controller {
   @tracked
   error = null;
 
+  @tracked isDeleting = false;
+
+  @action
+  onDeletePrompt() {
+    this.isDeleting = true;
+  }
+
+  @action
+  onCancel() {
```

Comment:
Suggestion: rename this to something more specific. We may have more functionality than just "delete" in the future, and `onCancel` would be ambiguous then.

Maybe `onDeleteCancel()`